src/locales/en/common.json

```diff
@@ -1501,7 +1501,8 @@
     "liquidityPoolsRemoveLiquidity": "{{value}} {{token}} removed from liquidity.",
     "liquidityPoolsStake": "{{value}} {{token}} staked.",
     "liquidityPoolsUnstake": "{{value}} {{token}} removed from liquidity.",
-    "liquidityPoolsClaimRewards": "{{value}} {{token}} claimed."
+    "liquidityPoolsClaimRewards": "{{value}} {{token}} claimed.",
+    "noActiveAccountFound": "No active account found. Please"
```

Comment:
This string seems unfinished, please complete the sentence or remove hanging "Please"

Reply:
Whoopsy! 👀